client/public/index.html

```diff
@@ -1,21 +1,21 @@
 <!DOCTYPE html>
 <html lang="">
   <head>
-    <meta charset='utf-8'>
-    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
-    <meta name='viewport' content='width=device-width,initial-scale=1.0'>
-    <script
-      src='https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v5.0.0/mapbox-gl-geocoder.min.js'></script>
-    <link href='https://api.mapbox.com/mapbox-gl-js/v2.8.2/mapbox-gl.css'
-          rel='stylesheet'>
-    <script
-      src='https://api.mapbox.com/mapbox-gl-js/v2.8.2/mapbox-gl.js'></script>
-    <link rel='icon' href='<%= BASE_URL %>favicon.ico'>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+    <script src="https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v5.0.0/mapbox-gl-geocoder.min.js"></script>
+    <link href="https://api.mapbox.com/mapbox-gl-js/v2.8.2/mapbox-gl.css" rel="stylesheet" />
+    <script src="https://api.mapbox.com/mapbox-gl-js/v2.8.2/mapbox-gl.js"></script>
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico" />
     <title>LeadOut</title>
   </head>
   <body>
     <noscript>
-      <strong>We're sorry but LeadOut doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
+      <strong
```

Comment:
can you change the formatting to 

<strong>
    ...
</strong>

this default line wrapping style for html is strange

Reply:
Haha yes this line wrapping was bizarre.